### DISCUSSION TITLE
How do I change the "Loading..." white page in my Next app?

### DISCUSSION THREAD

**Discussion by HarrisonHemstreet** [vercel/next.js#47984] [Help]
### Summary

Every time I load into my app, or change pages, a "Loading..." white page pops up. I would like to remove this page or change it in some way. I've searched my code for where this "Loading..." page is coming from, but have been unable to find it. Where should I look? Thank you!

**Comment by radosek** [comment]
You will need to create `loading.js` file.
https://beta.nextjs.org/docs/routing/loading-ui
